src/events.js

```diff
@@ -1,4 +1,25 @@
 export default [
+  {
+    title: "II Hackathon Kognitywistyczny Cog_HACK",
+    date: new Date(2020, 3, 28),
```

Comment:
Month argument range from 0 to 11. 
Hackathon will take palce in March, so I'd change `3 -> 2`